app/Models/Behaviors/HasAutoRelated.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Models\Behaviors;
+
+use App\Models\Vendor\Block;
+use Illuminate\Support\Str;
+
+trait HasAutoRelated
+{
+    public function related($id)
+    {
+        $modelName = Str::plural(Str::lower(class_basename(__CLASS__)));
+        ;
```

Comment:
Delete

Reply:
I was just stocking up in case a semicolon was missing somewhere 🥴

Reply:
you can have too few but not too many!